### DISCUSSION TITLE
Is this library still maintained?

### DISCUSSION THREAD

**Discussion by seogrady** [kobaltedev/kobalte#289] [General]
First off, great effort on Kobalte. Just like to know what the future direction for it is?  Any planned future releases?

**Comment by fabien-ml** [comment]
Hi, i'm sorry for the non communication for the past few months (was busy at work + personal problems).
I'm continuing to work on this library but at a lower pace.

The current roadmap for the lib is :
1) Remove the dependency of the i18n library used internally which is responsible of many issue and the large bundle size.
2) Fix the opened issues
3) Finish the `Calendar`/ `DatePicker` components

1 will be done before the end of the year, 2 and 3 will start in 2024

**Reply by seogrady** [reply]
Thanks for the update @fabien-ml 👍
